tests/RecurlyResource_Test.php

```diff
@@ -44,6 +44,22 @@ public function testFromJsonValidResource(): void
         $this->assertInstanceOf(\Recurly\Resources\TestResource::class, $result->getSingleChild());
     }
 
+    public function testCast(): void
```

Comment:
These tests could be added to `testFromJsonValidResource` as additional payload parameters.